app/javascript/components/auth-key-pair-cloud/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+import MiqFormRenderer from '../../forms/data-driven-form';
+import createSchema from './auth-key-pair-cloud-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import { API } from '../../http_api';
+
+
+const AuthKeypairCloudForm = () => {
+  const [fields] = useState([]);
```

Comment:
You don't really need this, the fields are never changing, so it's okat to have it directly without the state.